docgen/src/guides/migration.md

```diff
@@ -88,6 +88,16 @@ As of now, we consider the engine used to build the widgets in InstantSearch.js
 as an implementation detail. Since we do not expose it anymore, we'll be able
 to change it and use the best solution for each release.
 
+## RangeSlider widget is using Rheostat as Slider Component
+
+Since we had unclosed issues with the old slider, we switched to a new
```

Comment:
I would rephrase this more like `it had more features and we want to use it`, now it’s a bit negative